ros/angel_system_nodes/angel_system_nodes/spatial_mapper.py

```diff
@@ -251,8 +225,10 @@ def detection_callback(self, detection):
             for p in bounds_screen_pos:
                 scaled_point = self.scale_pixel_coordinates(p)
 
-                focal_length_x = projection_matrix_2d[0][0]
-                focal_length_y = projection_matrix_2d[1][1]
+                #focal_length_x = projection_matrix[0][0]
+                focal_length_x = 1.63
+                #focal_length_y = projection_matrix[1][1]
+                focal_length_y = 2.50
```

Comment:
We should add here an attention grabbing comment to highlight that this is where the hack is. Probably also at the top in the class doc-string.,